tests/requirements.txt

```
@@ -1,4 +1,4 @@
 pytest
 pytest-cov
-conan==1.62.0
-velocitas-lib==0.0.4
+conan>2
```

Huh, your're future oriented ^^ But maybe we should stick to version2, for now:
```suggestion
conan==2.*
```

---

yeah, didn't think that conan 3 will be released soon :D